src/pages/groupComparison/styles.module.scss

```diff
@@ -97,3 +97,43 @@
         margin-right: 30px;
     }
 }
+.survivalPlotHeaderContainer {
+    display: flex;
+    margin-right: 5px;
+}
+
+.survivalPlotHeader {
```

Comment:
not certain but i think you can use existing pill styles from bootstrap.  check plots tab implementation

Reply:
will fix in another pr

Reply:
I'll create a new pr after merging. Have some code changes based on this pr https://github.com/leexgh/cbioportal-frontend/commit/8d647e93f9a5e5e963404581da3e4ac0cd529f23